BridgeSDK/BridgeSDK.h

```diff
@@ -90,6 +97,22 @@ static SBBEnvironment gDefaultEnvironment = kDefaultEnvironment;
 + (void)setupWithStudy:(NSString *)study useCache:(BOOL)useCache environment:(SBBEnvironment)environment;
 
 /*!
+ * Set up the Bridge SDK for the given study and server environment. Usually you would only call this version
```

Comment:
Re: this code comment only calling this version for testing, there should be another method with the same signature but without the environment parameter, which calls this one with the default environment (see the other setupWithStudy method pairs), which is what would be called from the app to set up Bridge with non-default cache timespans.